packages/openblas/patches/0003-fix-cmake-ar-use.patch

```diff
@@ -0,0 +1,24 @@
+Index: openblas-0.3.23/CMakeLists.txt
+===================================================================
+--- openblas-0.3.23.orig/CMakeLists.txt
++++ openblas-0.3.23/CMakeLists.txt
+@@ -252,15 +252,15 @@ if (APPLE AND DYNAMIC_ARCH AND BUILD_SHA
+   if (NOT NOFORTRAN)
+   set (CMAKE_Fortran_USE_RESPONSE_FILE_FOR_OBJECTS 1)
+   set (CMAKE_Fortran_CREATE_SHARED_LIBRARY
+- "sh -c 'cat ${CMAKE_BINARY_DIR}/CMakeFiles/openblas_shared.dir/objects*.rsp | xargs -n 1024 ar -ru libopenblas.a && exit 0' "
+- "sh -c 'ar -ru libopenblas.a ${CMAKE_BINARY_DIR}/driver/others/CMakeFiles/driver_others.dir/xerbla.c.o && exit 0' "
++ "sh -c 'cat ${CMAKE_BINARY_DIR}/CMakeFiles/openblas_shared.dir/objects*.rsp | xargs -n 1024 emar -ru libopenblas.a && exit 0' "
```

Comment:
I guess we can use `${CMAKE_AR}` here. Maybe we should upstream this.

```suggestion
+ "sh -c 'cat ${CMAKE_BINARY_DIR}/CMakeFiles/openblas_shared.dir/objects*.rsp | xargs -n 1024 ${CMAKE_AR} -ru libopenblas.a && exit 0' "
```

Reply:
yeah, good point